### ISSUE TITLE
[Translate] Settings

### ISSUE THREAD

**Issue by xbvrhlmb** [oxen-io/session-desktop#1868]
I have already made a report, but I highlighted everything in one list, something was fixed, something was missed. So I duplicated with the new information.

![session_settings](https://user-images.githubusercontent.com/43858033/129933590-ca788dba-2319-4839-a12a-25ab204cb6e4.png)

![session_settings2](https://user-images.githubusercontent.com/43858033/129933593-a5e70d28-ea0d-46c2-a9ff-4c50d4bf01c0.png)

**Comment by Bilb** [comment]
https://crowdin.com/project/session-desktop

**Comment by xbvrhlmb** [comment]
> https://crowdin.com/project/session-desktop

really? 
I have highlighted those items that are not in the list for translation.

![Screenshot 2021-08-19 at 14-27-20 Session Desktop — Translation Project on Crowdin](https://user-images.githubusercontent.com/43858033/130061099-14a34361-2c27-4bab-8d82-d6222ded4999.png)

**Comment by Bilb** [comment]
Hey @xbvrhlmb thanks for the heads up.
I've just checked, and it just looks like you finished the russian translation but the latest release did not use it yet. I'll include in the next release.

Here is some screenshot I took on my dev

![image](https://user-images.githubusercontent.com/1544279/130540996-3de9818d-ed03-46bc-9171-520e2475a58e.png)
![image](https://user-images.githubusercontent.com/1544279/130541055-af223043-655f-4093-a8e2-ea7ff442539c.png)
